ls_pre_commit_hooks/circleci-config-validate.sh

```diff
@@ -28,8 +49,21 @@ if ! command -v circleci &>/dev/null; then
   exit 1
 fi
 
-if ! eMSG=$(circleci --skip-update-check config validate -c "${_PATH}"); then
-  echo "CircleCI Configuration Failed Validation."
-  echo $eMSG
-  exit 1
-fi
+for path in "${positional_args[@]}"
+do
+
+  cmdArgs=('--skip-update-check' 'config' 'validate' '-c' "${path}")
+  if [ -n "${_ORG_SLUG}" ]; then
+    cmdArgs+=('-o' "${_ORG_SLUG}")
+  fi
+
+  if ! eMSG=$(circleci "${cmdArgs[@]}"); then
```

Comment:
This actually doesn't catch the output properly, but it never did. So later we don't echo the useful `circleci setup` tip. I'll be back to this.

Reply:
```suggestion
  if ! eMSG=$(circleci "${cmdArgs[@]}" 2>&1); then
```

Probably need to redirect stderr as well?

Reply:
Works! That was easy, thanks ;)